OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -816,22 +817,14 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="AfterBuild">
-    <MakeDir Directories="$(SolutionDir)mods/common/"/>
```

Comment:
Looks like some unwanted VS reformatting slipped in. Can you please unstage these?